popde/density_estimate.py

```diff
@@ -1,3 +1,97 @@
+class SimpleKernelDensityEstimation:
+    """
+    Fit and evaluate general Kernel Density Estimation (KDE) 
+
+    Methods:
+    --------
+    fit(data, bandwidth=None, kernel=None):
+        Fit the KDE to new data.
+
+    evaluate(points):
+        Evaluate the KDE at given data points.
+
+    Examples:
+    ---------
+    # Create a KDE instance and fit it to data
+    data = np.random.randn(100)
```

Comment:
this example is only 1d data.  I think we should have an example that uses multi-dimensional data, eg 2d
Also, the example code does `kde.fit()` which doesn't exist.

Reply:
fixed this